libethashseal/genesis/mainNetwork.cpp

```diff
@@ -47,10 +47,10 @@ R"E(
         "0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
         "0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } },
         "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp", "startingBlock": "0x42ae50" } },
-        "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "startingBlock": "0x42ae50", "linear": { "base": 500, "word": 0 } } },
```

Comment:
So where is the new cost encoded then?

Reply:
It's now in the code, because it needs the current block number to figure out the precompile cost https://github.com/ethereum/aleth/blob/403dc9067a453420c43c974f919586285dc30ab7/libethcore/Precompiled.cpp#L181